app/uk/gov/hmrc/thirdpartyapplication/connector/AwsApiGatewayConnector.scala

```diff
@@ -36,12 +36,18 @@ class AwsApiGatewayConnector @Inject()(http: HttpClient, config: AwsApiGatewayCo
   val awsApiKey: String = config.awsApiKey
   val apiKeyHeaderName = "x-api-key"
 
+  private def updateUsagePlanURL(rateLimitTier: RateLimitTier): String = s"${config.baseUrl}/v1/usage-plans/$rateLimitTier/api-keys"
+  private def deleteAPIKeyURL(applicationName: String): String = s"${config.baseUrl}/v1/api-keys/$applicationName"
+
   def createOrUpdateApplication(applicationName: String, upsertApplicationRequest: UpsertApplicationRequest)(hc: HeaderCarrier): Future[HasSucceeded] = {
```

Comment:
I don't think we need the UpsertApplicationRequest type anymore. We could just pass in the server token instead and delete that type.

Reply:
Done